### PR TITLE
Update docs to reflect message board migration

### PR DESCRIPTION
**Pull request by ennsharma**

### CODE REVIEW

File: src/tutorials/messaging.md [hunk 1]
```diff
@@ -3,11 +3,12 @@
 In this tutorial we will create a private, uncensorable messaging service.
 This tutorial demonstrates intermediate concepts: events and composed types.
 
+**tl;dr:** The code for this example can be found [here](https://github.com/oasislabs/tutorials/tree/master/message-board).
```
Comment (ssavvides) [inline src/tutorials/messaging.md]: This is 404

Comment (ssavvides) [reply src/tutorials/messaging.md]: also this: https://github.com/oasislabs/oasis-rs/issues/215

**Comment by ravenac95** [comment]
@ennsharma @nhynes can we close this? this also seems outdated?

**Comment by ravenac95** [comment]
@ennsharma is this still relevant?

**Comment by ravenac95** [comment]
@ennsharma Closing due to inactivity

**Pull request closed without merging**